src/components/QueryEditor/MetricAggregationsEditor/SettingsEditor/index.tsx

```diff
@@ -28,6 +28,8 @@ interface Props {
   previousMetrics: MetricAggregation[];
 }
 
+const orderLabel = 'Test Order select Container';
```

Comment:
I don't think this needs to be a separate variable

Reply:
Removed this altogether, since afaik InlineField doesn't need an explicit arial-label